my-app/src/App.css

```diff
@@ -0,0 +1,112 @@
+* {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  /* font-size: 12px; */
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  background: #ffffff;
+  font-family: "Lato", sans-serif;
+  padding: 8px 8px;
+  margin: 0 24px;
+  color: #37474f;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.post-title {
+  color: #37474f;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 0;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.post-url {
+  font-size: 12px;
+  margin-left: 8px;
+  color: #0277bd;
+  line-height: 24px;
+}
+
+.post-points {
+  font-size: 10px;
+  margin: 0 8px 0 0;
+}
+
+.post-second-line-item {
+  font-size: 10px;
+  margin: 0 8px 0 0;
+  color: #0277bd;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.btn {
+  box-shadow: none;
+  font-family: "Lato", sans-serif;
+  border: none;
+  background: #f4511e;
+  border-radius: 4px;
+  padding: 0 32px;
+  display: inline-block;
+  margin: 16px 0 40px 32px;
+  height: 40px;
+  line-height: 40px;
+  color: #ffffff;
+  text-transform: uppercase;
+  outline: none;
+  -webkit-transition: all 0.3s;
+  transition: all 0.3s;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.App {
+  text-align: left;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
```

Comment:
:+1: for the CSS3 stuff.

Just curious: why the dynamic / relative font size in the header, but an absolute font-size in the body?

Reply:
At this point, I was grabbing font-sizes directly from dev tools in the example. However, I do realize that em/rem get computed to px in the DOM so I could have made sure to still use relative font-size based on the px from the example.